src/Pos/DB/Block/Block.hs

```diff
@@ -0,0 +1,302 @@
+{-# LANGUAGE ScopedTypeVariables #-}
+
+-- | Interface to Blocks DB.
+
+module Pos.DB.Block.Block
+       ( getBlock
+       , getBlockHeader
+       , getUndo
+       , getBlockWithUndo
+       , putBlund
+
+       , prepareBlockDB
+
+       -- * Load data
+       , loadBlundsWhile
+       , loadBlundsByDepth
+       , loadBlocksWhile
+       , loadHeadersWhile
+       , loadHeadersByDepth
+       , loadHeadersByDepthWhile
+       ) where
+
+import           Control.Lens              (_Wrapped)
+import           Control.Monad.Trans.Maybe (MaybeT (MaybeT), runMaybeT)
```

Comment:
Yes, somehow `MaybeT` didn't get into `mtl` library, so should be imported from `transformers` package. Unless we don't mind depend on some strange [`MaybeT`](http://hackage.haskell.org/package/MaybeT) package. But now I thought that it makes sense to export `MaybeT` from `universum`. Don't see reasons why not do it. @neongreen @pva701 what do you think?